### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by zags**
#### What are the relevant tickets?
#16 

#### What's this PR do?
This adds unit tests for views and models and fixes some bugs encountered in the process

### CODE REVIEW

**Comment by pdpinch** [comment]
This PR passed tests, but I'm getting a 500 from the deployment.

File: sga/tests/models_test.py [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+    Test that methods on models work as expected
+    """
+
+    def test_timestampedmodel_update(self):
+        """
+        Tests the .update() method on TimeStampedModel
+        """
+        course = self.get_test_course()  # Course is a subclass of TimeStampedModel
+        initial_updated_on = course.updated_on
+        sleep(0.1)
```
Comment (giocalitri) [inline sga/tests/models_test.py]: why do you need this?

**Comment by giocalitri** [comment]
@pdpinch I think this is not related to the code itself: the problem was there also on other PRs and @blarghmatey took a look last week

**Comment by giocalitri** [comment]
General comment: I think it will be better to remove the `# pragma: no cover`: I think we are not aiming to the 100% coverage if it is not obtained by real tests.

**Comment by giocalitri** [comment]
only a couple of comments, but overall looks fine to me

**Pull request merged**